e2e-tests/playwright/e2e/plugins/rbac/rbac-as-admin.spec.ts

```diff
@@ -0,0 +1,258 @@
+/* eslint-disable no-empty-pattern */
+import test, { expect, Page } from "@playwright/test";
+import { UIhelperPO } from "../../../support/pageObjects/global-obj";
+import {
+  RoleListPO,
+  RoleFormPO,
+  HomePagePO,
+  RoleOverviewPO,
+} from "../../../support/pageObjects/page-obj";
+import { Common, setupBrowser } from "../../../utils/Common";
+import { UIhelper } from "../../../utils/UIhelper";
+import { Roles } from "../../../support/pages/rbac";
+
+type RbacAsAdminFixture = {
+  uiHelper: UIhelper;
+  rolesHelper: Roles;
+  testId: { testId: string; testRole: string; composedRole: string };
+};
+
+let myPage: Page;
+
+const base = test.extend<RbacAsAdminFixture>({
+  //prettier-ignore
+  uiHelper: async ({}, use) => { //NOSONAR
+    const uiHelper = new UIhelper(myPage);
+    await myPage.goto("/rbac");
+    await use(uiHelper);
+  },
+  //prettier-ignore
+  rolesHelper: async ({}, use) => { //NOSONAR
+    const rolesHelper = new Roles(myPage);
+    await use(rolesHelper);
+  },
+  //prettier-ignore
+  testId: async ({}, use) => { //NOSONAR
+    const testId = Date.now().toString();
+    const testRole = `test-role-${testId}`;
+    const composedRole = `role:default/${testRole}`;
+    await use({ testId, testRole, composedRole });
+  },
+});
+
+base.describe.serial("Test RBAC plugin as an admin user", () => {
+  base.beforeAll(async ({ browser }, testInfo) => {
+    myPage = (await setupBrowser(browser, testInfo)).page;
+    await new Common(myPage).loginAsGithubUser();
+  });
+
+  base.beforeEach(
+    async () => await new Common(myPage).checkAndClickOnGHloginPopup(),
```

Comment:
checkAndClickOnGHloginPopup isn't required unless the plugin requires github user login

Reply:
Isn't this the case? We are doing Github Login in the set

Reply:
* or do you mean that github login is not required for these tests?

Reply:
checkAndClickOnGHloginPopup was meant for plugins that require additional login apart from backstage login even though you logged in with github it still asks. like github-actions, issues, pr/mr are few plugins that require this.

Reply:
I think this is also necessary in some other cases, like happened here:
https://redhat-internal.slack.com/archives/C04CUSD4JSG/p1730884919450929?thread_ts=1730730693.542309&cid=C04CUSD4JSG
But I am not quite sure, I'll try to remove it